lib/survicate_sdk.dart

```diff
@@ -89,9 +91,13 @@ class SurvicateSdk {
   /// Sets multiple user traits at once using the provided [traits] map.
   /// See also:
   ///  * [setUserTrait], the method used to set a single trait.
-  static void setUserTraits(Map<String, String> traits) async {
+  static void setUserTraits(List<UserTrait> traits) async {
```

Comment:
Update docs comment: "Sets multiple user traits at once using the provided [traits] **list**"